doc/man3/RAND_DRBG_new.pod

```diff
@@ -151,13 +149,17 @@ cryptographic routines that obtain random data directly or indirectly.
 
 =head1 SEE ALSO
 
-L<OPENSSL_zalloc(3)>,
-L<OPENSSL_secure_zalloc(3)>,
 L<RAND_DRBG_generate(3)>,
 L<RAND_DRBG(7)>
 
 =head1 HISTORY
 
+The RAND_DRBG_set() function was deprecated in OpenSSL 3.0.
+
+The distinction between secure heap allocated DRBG and heap allocated DRBG was
```

Comment:
Did this distinction exists in 1.1.1? I thought RAND_DRBG_secure_new() did not exist there?

Reply:
No it doesn't exist in 1.1.1.  Removing these words.